### DISCUSSION TITLE
Need to add more detail description inside snippet description like HTML content link and table

### DISCUSSION THREAD

**Discussion by anishkumar-bhut** [microsoft/vscode-discussions#1835] [Extension Development QnA] [answered]
I see, we can add  scope, prefix, body, description in description it just a string but i want to add detail description and link to redirect another page for some html content so does it possible?

**Comment by isidorn** [comment, marked as answer]
https://code.visualstudio.com/docs/editor/userdefinedsnippets

https://code.visualstudio.com/api/language-extensions/snippet-guide

Should help
